Makefile

```diff
@@ -137,11 +137,18 @@ reproducible-prestate:   ## Builds reproducible-prestate binary
 	make -C ./op-program reproducible-prestate
 .PHONY: reproducible-prestate
 
-cannon-prestate: op-program cannon ## Generates prestate using cannon and op-program
+# Checks if prestate outputs are missing
+cannon-prestate: op-program/bin/prestate-proof.json
+.PHONY: cannon-prestate
+
+op-program/bin/prestate-proof.json:
```

Comment:
I think this specifically needs to declare a dependency on every file in op-program and all of its transitive dependencies like the derive package. Right now if op-program/bin/prestate-proof.json exists it assumes its up to date which is wrong.